### PR TITLE
GH-17: create settings form

### LINKED ISSUE #17: Create "Settings" form
Create a form for the "Settings" page. [Figma reference ](https://www.figma.com/file/KT1vLyYfL28NZDdNeopdNj/re-design-hitt?node-id=569%3A10941)

### PR DESCRIPTION
**Pull request by alex-waverley**
Robot settings form (UI + logic) was added to the web app

Closes #17

### CODE REVIEW

**Comment by sskorol** [comment]
I believe it closes #17

File: frontend/src/store/Settings/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    speed_min: 0,
+    speed_max: 1,
+    speed_step: 20
+  },]
+
+  @action
+  updateRobotSettings = (robot_data) => {
+    // TODO: send robot_data to the beckend when it is ready
+    this.robots_settings = this.robots_settings.map((robot) => {
+      if(robot.id === robot_data.id) {
```
Comment (sskorol) [inline frontend/src/store/Settings/index.js:40]: You can use ternary here

File: frontend/src/components/roboTypeIcon/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+</SvgIcon>
+
+const RobotTypeIcon = (props) => {
+  let Icon;
+
+  switch (props.type) {
+    case 'Legged':
```
Comment (sskorol) [inline frontend/src/components/roboTypeIcon/index.js:16]: You may want to create an enum for better reusability.

File: frontend/src/store/Settings/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  }
+
+  @observable widgets = [
+    { label: 'Screen', id: '1', selected: true },
+    { label: 'Battery', id: '2', selected: true },
+    { label: 'Robot\'s Speed', id: '3', selected: true },
+    { label: 'Additional actions', id: '4', selected: true },
+  ]
+
+  @observable robotTypes = [
```
Comment (sskorol) [inline frontend/src/store/Settings/index.js:15]: Just to clarify: is it a stub for the data that should come from BE?

**Review by sskorol — CHANGES_REQUESTED** [review]

File: frontend/src/store/Settings/index.js (hunk 3 above)
Comment (alex-waverley) [reply frontend/src/store/Settings/index.js:15]: correct, it is a stub

File: frontend/src/components/roboTypeIcon/index.js (hunk 2 above)
Comment (alex-waverley) [reply frontend/src/components/roboTypeIcon/index.js]: done

File: frontend/src/store/Settings/index.js (hunk 1 above)
Comment (alex-waverley) [reply frontend/src/store/Settings/index.js]: done

**Review by sskorol — APPROVED** [review]

**Comment by alex-waverley** [comment]
@sskorol commits have been squashed

**Pull request merged**